sphinx.go

```diff
@@ -525,6 +526,14 @@ func WithBlindingPoint(point *btcec.PublicKey) ProcessOnionOpt {
 	}
 }
 
+// WithTLVPayloadOnly is a functional option that signals that the onion packet
```

Comment:
meaning the onion packet here is the `onion_message_packet ` using spec vocabulary